### DISCUSSION TITLE
Why not add admin pages dynamically rather than adding them in database as Alias?

### DISCUSSION THREAD

**Discussion by NishchayChauhan251980** [oqtane/oqtane.framework#2423] [Q&A]
Hi @sbwalker ,

While looking at the Entries in Alias database, I realize that admin pages alias are being added for each site.

I believe this should be pretty much possible to handle all admin routes dynamically?

We have a situation where we created let's say just 5 pages  for each blog site in resultant the Alias Table is going to have some 25 entries.

Another enhancement that we feel should be addressed is capability to search Sites and Site Management and Pages under Pages Management.

Thank you so much!

**Comment by sbwalker** [comment]
@NishchayChauhan251980 The Alias table has nothing to do with Pages - it should only contain the root Urls which map to a specific Site. For example if you have a Site which is accessed at www.mysite.com then you will only have 1 entry in the Alias table with a Name of "www.mysite.com" - and it does not matter how many Pages exist in your Site, you will still only have 1 Alias. Can you please explain how to duplicate the issue you are experiencing as I cannot reproduce this behavior.
